ocs/ocp.py

```diff
@@ -134,23 +148,48 @@ def apply(self, yaml_file):
         command = f"apply -f {yaml_file}"
         return self.exec_oc_cmd(command)
 
-    def wait_for_resource_status(
-        self, resource_name, condition, timeout=30, sleep=3
+    def wait(
+        self, condition, resource_name='', selector=None, resource_count=0,
+        timeout=60, sleep=3
```

Comment:
Missing documentation for timeout and sleep

Reply:
fixed